### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by khushikdu**
High-Resolution View
Screenshot from 2024-02-15 14-26-57

Desktop-View
image

Tablet View
Screenshot from 2024-02-15 13-49-32

https://video.drift.com/v/abJvrhmUghii2vcvuyOfo6dWeRakWCmpk1v0YAqXRw/

### CODE REVIEW

**Comment by khushikdu** [comment]
| Metric | Value |
| ------ | ----- |
| alert_status | <b>:white_check_mark: OK</b> |
| bugs | 4 |
| code_smells | 6 |
| reliability_rating | D (at least 1 Critical Bug)<br/>Note: A being best, E being worst |
| security_rating | A (0 Vulnerabilities)<br/>Note: A being best, E being worst |
| vulnerabilities | 0 |

**Comment by Nitesh2503** [comment]
Always add proper commit messages
Follow this message syntax `<branch_namev : "Commit message (a bit descriptive)">`
And if too many commits are there then squash them

File: frontend/assessments/1/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="main">
+
```
Comment (Nitesh2503) [inline frontend/assessments/1/index.html:11]: Format the code properly

File: frontend/assessments/1/index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <header id="top-header">
```
Comment (Nitesh2503) [inline frontend/assessments/1/index.html:12]: The class names must be proper and intuitive

File: frontend/assessments/1/index.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            <div id="ul-bar">
+                <div class="nav-items first">Home</div>
+                <div class="nav-items">About</div>
+                <div class="nav-items">Service</div>
+                <div class="nav-items">Contact</div>
+            </div>
+        </nav>
+
+        <div id="tagline">
+            <p><b>
```
Comment (Nitesh2503) [inline frontend/assessments/1/index.html:26]: Avoid using `b` tag instead use css for this

File: frontend/assessments/1/style.css [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+  font-family: "Work sans";
+}
+
+#main #top-header {
+  background-color: rgb(255, 179, 80);
+  padding: 1rem;
+}
+#main #top-header .logo {
+  font-size: 50px;
+  color: white;
```
Comment (Nitesh2503) [inline frontend/assessments/1/style.css:15]: Follow the consistent colour formats

File: frontend/assessments/1/style.css [hunk 5]
```diff
@@ -0,0 +1,215 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  box-sizing: border-box;
+  font-family: "Work sans";
+}
+
+#main #top-header {
+  background-color: rgb(255, 179, 80);
```
Comment (Nitesh2503) [inline frontend/assessments/1/style.css:10]: Store the colors/hard-coded values as the constants in different file and import it

File: frontend/assessments/1/style.css [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+  color: white;
+  line-height: 170%;
+}
+#main #footer {
+  color: white;
+}
+#main #footer #footer-data {
+  background-color: rgb(74, 73, 73);
+  margin: 2px;
+  position: sticky;
```
Comment (Nitesh2503) [inline frontend/assessments/1/style.css:78]: For footer use position: fixed

File: frontend/assessments/1/style.css [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+  margin-bottom: 0;
+  padding: 1rem;
+  text-align: center;
+}
+
+@media screen and (min-width: 767px) and (max-width: 1367px) {
```
Comment (Nitesh2503) [inline frontend/assessments/1/style.css:84]: Add the proper comment for which view this media query is

File: frontend/assessments/1/style.scss [hunk 8]
```diff
@@ -0,0 +1,240 @@
...
+            
+            .nav-items{
+                
+                padding: 1rem;
+            }
+        }
+    }
+    #tagline{
+        background-image: url('Re-Assessment Asset.jpg');
+        // height: 50rem;
```
Comment (Nitesh2503) [inline frontend/assessments/1/style.scss:38]: Don't push the commented/stale code

File: frontend/assessments/1/style.scss [hunk 9]
```diff
@@ -0,0 +1,240 @@
...
+    #bottom{
+        display: flex;
+        flex-direction: column;
+        background-color: rgb(232, 214, 214);
+        text-align: justify;
+        color: rgb(58, 54, 54);
+        #welcome{
+            margin: 4rem ;
+            padding-left: 2.5rem;
+            margin-bottom: 0;
```
Comment (Nitesh2503) [inline frontend/assessments/1/style.scss:217]: Optimize to cover 1st and 2nd css property in single line

File: frontend/assessments/1/style.scss [hunk 10]
```diff
@@ -0,0 +1,240 @@
...
+            #headtext{
+                line-height: 150%;
+                padding-bottom: 1rem;
+            }
+            #body-text{
+                line-height: 150%;
+            }
+        }
+        #left-items{
+
```
Comment (Nitesh2503) [inline frontend/assessments/1/style.scss:228]: Format the code properly

File: frontend/assessments/1/style.scss [hunk 11]
```diff
@@ -0,0 +1,240 @@
...
+                line-height: 150%;
+                padding-bottom: 1rem;
+            }
+            #body-text{
+                line-height: 150%;
+            }
+        }
+        #left-items{
+            margin: 1rem;
+            margin-right: 4rem;
```
Comment (Nitesh2503) [inline frontend/assessments/1/style.scss:70]: Optimize this

File: frontend/assessments/1/style.scss [hunk 12]
```diff
@@ -0,0 +1,240 @@
...
+            padding: 1rem 0.5rem 1rem 0.5rem;
+            background-color: rgb(74, 73, 73);
+            height: max-content;
+            color: white;
+            line-height: 170%;
```
Comment (Nitesh2503) [inline frontend/assessments/1/style.scss:75]: Avoid this

File: frontend/assessments/1/style.scss [hunk 13]
```diff
@@ -0,0 +1,240 @@
...
+                color: white;
+                line-height: 170%;
+            }
+        }
+    }
+    
+}
+
+
+@media screen and (max-width: 768px)  {    #main{
```
Comment (Nitesh2503) [inline frontend/assessments/1/style.scss:166]: Add comments

**Pull request merged**